<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="resolutionGroupBox">
          <property name="title">
           <string>Разрешение</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QSpinBox" name="size_x">
             <property name="suffix">
              <string> pix</string>
             </property>
             <property name="prefix">
              <string>x = </string>
             </property>
             <property name="maximum">
              <number>3840</number>
             </property>
             <property name="value">
              <number>800</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="size_y">
             <property name="suffix">
              <string> pix</string>
             </property>
             <property name="prefix">
              <string>y = </string>
             </property>
             <property name="maximum">
              <number>2160</number>
             </property>
             <property name="value">
              <number>600</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="zoomGroupBox">
          <property name="title">
           <string>Масштаб</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QDoubleSpinBox" name="coef">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="prefix">
              <string>2^</string>
             </property>
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="minimum">
              <double>-5.000000000000000</double>
             </property>
             <property name="maximum">
              <double>40.000000000000000</double>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::DefaultStepType</enum>
             </property>
             <property name="value">
              <double>-3.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="stepsGroupBox">
          <property name="title">
           <string>макс. число шагов</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QSpinBox" name="steps">
             <property name="minimum">
              <number>1000</number>
             </property>
             <property name="maximum">
              <number>1000000</number>
             </property>
             <property name="singleStep">
              <number>1000</number>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="placeGroupBox">
          <property name="title">
           <string>координаты</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QDoubleSpinBox" name="coord_x">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="prefix">
              <string>x = </string>
             </property>
             <property name="decimals">
              <number>16</number>
             </property>
             <property name="minimum">
              <double>-2.000000000000000</double>
             </property>
             <property name="maximum">
              <double>2.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="coord_y">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="prefix">
              <string>y = </string>
             </property>
             <property name="decimals">
              <number>16</number>
             </property>
             <property name="minimum">
              <double>-2.000000000000000</double>
             </property>
             <property name="maximum">
              <double>2.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="raster_label" name="image">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="refresh_button">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resample_button">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;выставляет координаты&lt;/p&gt;&lt;p&gt;в соответствии с фактическим&lt;/p&gt;&lt;p&gt;размером области изображения&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>подогнать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_image">
        <property name="text">
         <string>сохранить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="julia">
        <property name="text">
         <string>Множество
Жюлиа</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>raster_label</class>
   <extends>QLabel</extends>
   <header>raster_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
